.github/workflows/ci.yml

```diff
@@ -14,6 +14,9 @@ on:
       - "*"
       - "*/*"
 
+env:
```

Comment:
note to reviewers: only changes in this file (ci.yml) are relevant.  Everything else can be ignored in this review.